engine/common/splitter/network/example_test.go

```diff
@@ -6,9 +6,10 @@ import (
 
 	"github.com/rs/zerolog"
 
+	"github.com/onflow/flow-go/network/channels"
+
```

Comment:
nit: any reason why this is separated by a blank line from the main block of local imports in all of the files? Seems like a configuration issue with your editor

Reply:
When refactoring (moving channels -> channels package) Goland does not sort the imports as it updates them. I've went through and fixed them.